universal-application-tool-0.0.1/app/views/BaseHtmlView.java

```diff
@@ -17,6 +27,64 @@
  */
 public abstract class BaseHtmlView {
 
+  public Tag renderHeader(String headerText) {
+    return h1(headerText);
+  }
+
+  protected ImmutableList<DomContent> textInputWithLabel(
+      String labelValue, String inputId, Optional<String> value) {
+    Tag labelTag = label(labelValue).attr("for", inputId);
```

Comment:
suggestion: prefer to use `Attr` constants: https://javadoc.io/static/com.j2html/j2html/1.2.1/j2html/attributes/Attr.html

Reply:
Done.

Reply:
Done.